schema.py

```diff
@@ -139,3 +163,84 @@ def resolve_eateries(self, info, eatery_id=None):
 
   def resolve_operating_hours(self, info, eatery_id=None):
     print(list(Data.operating_hours.values())[0][0].date)
+
+  def resolve_account_info(self, info, session_id=None):
+    if session_id is None:
+      return "Provide a valid session ID!"
+
+    account_info = {}
+
+    # URLs to send requests to
+    user_url = 'https://services.get.cbord.com/GETServices/services/json/user'
+    commerce_url = 'https://services.get.cbord.com/GETServices/services/json/commerce'
+
+    # Query 1: Get user id
+    user_id = requests.post(
+        user_url,
+        json={
+            'version': '1',
+            'method': 'retrieve',
+            'params': {
+                'sessionId': session_id
+            }
+        }
+    ).json()['response']['id']
+
+    # Query 2: Get finance info
+    accounts = requests.post(
+        commerce_url,
+        json={
+            'version': '1',
+            'method': 'retrieveAccountsByUser',
+            'params': {
+                'sessionId': session_id,
+                'userId': user_id
+            }
+        }
+    ).json()['response']['accounts']
+
+    for acct in accounts:
+      if acct['accountDisplayName'] == ACCOUNT_NAMES['citybucks']:
+        account_info['cityBucks'] = str(acct['balance'])
+      if acct['accountDisplayName'] == ACCOUNT_NAMES['laundry']:
+        account_info['laundry'] = str("{0:.2f}".format(round(acct['balance'],2)))
+      if acct['accountDisplayName'] == ACCOUNT_NAMES['brbs']:
+        account_info['brbs'] = str(acct['balance'])
+      # Need more research to implement swipes:
+      # Each plan has a different accountDisplayName
+      account_info['swipes'] = ''
+
+    # Query 3: Get list of transactions
+    CORNELL_INSTITUTION_ID = '73116ae4-22ad-4c71-8ffd-11ba015407b1'
```

Comment:
put in constants.py

Reply:
@henrytso this should be removed from here?